### PR TITLE
build(lint): migrate from tslint to eslint

### PR DESCRIPTION
**Pull request by sergeymakoveev**
1. migrate from tslint to eslint
2. fix linter errors

### CODE REVIEW

**Comment by sutarmin** [comment]
Don't we want to remove anything related to `tslint`? We are _migrating_, right?

**Comment by sergeymakoveev** [comment]
tstint is now used by many developers and CI, so I did not delete tstint and its dependencies.

**Comment by raveclassic** [comment]
Let's keep only one solution

**Comment by sergeymakoveev** [comment]
> Let's keep only one solution

Done.

File: packages/lint/package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "module": "dist",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "dx-tools build-lib src dist --failOnError",
-    "clean": "dx-tools clean dist",
+    "build": "dx-tools build-lib src dist --failOnError | exit 0",
```
Comment (raveclassic) [inline packages/lint/package.json]: How does this change affect build?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (raveclassic) [inline .eslintrc.js]: Is it possible to use json?

File: packages/lint/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
+module.exports = {
```
Comment (raveclassic) [inline packages/lint/.eslintrc.js]: Can we use json?

File: packages/lint/package.json (hunk 1 above)
Comment (sergeymakoveev) [reply packages/lint/package.json]: After removing packages/lint/src/disallowImportsRule.ts in lint-package his "src"-folder became empty.
After removing, `yarn build`,  `yarn clean` and `yarn prepare` in linter-package return errors.
Therefore, the execution of the script `yarn lerna run prepare` in the main-package also return an error.
We can remove `build`,  `clean` and `prepare` in linter-package, but we anyway rollback this changes in #162

Now I override calling `build` and `clean` as success with forced return 0.
I make it for save build/prepare processes "as is".
In #162 we will rollback this override.

We will remove or save `build`,  `clean` and `prepare` scripts in linter-package?

Comment (raveclassic) [reply packages/lint/package.json]: I would keep the package as clean as possible. If we need any removed code/configuration further we can always pick it from git history.

File: .eslintrc.js (hunk 2 above)
Comment (sergeymakoveev) [reply .eslintrc.js]: Yes.

File: packages/lint/.eslintrc.js (hunk 3 above)
Comment (sergeymakoveev) [reply packages/lint/.eslintrc.js]: Yes.
Rename it to .json?

Comment (raveclassic) [reply packages/lint/.eslintrc.js]: yep

File: packages/lint/package.json (hunk 1 above)
Comment (sergeymakoveev) [reply packages/lint/package.json]: Done.

File: packages/lint/.eslintrc.js (hunk 3 above)
Comment (sergeymakoveev) [reply packages/lint/.eslintrc.js]: Done.

**Pull request merged**